Observer/Frontend/CustomerRegistered.php

```diff
@@ -30,6 +30,9 @@ public function __construct(
 
     public function execute(Observer $observer)
     {
+        $requestParams = $_REQUEST;
```

Comment:
Please check the $customer object. I feel this information should be read via observer->getData (customer) flow